pkg/kubelet/apis/config/types.go

```diff
@@ -440,10 +440,13 @@ type KubeletConfiguration struct {
 	// is true and upon the initial registration of the node.
 	// +optional
 	RegisterWithTaints []v1.Taint
-
 	// registerNode enables automatic registration with the apiserver.
 	// +optional
 	RegisterNode bool
+	// Tracing specifies the options of tracing.
```

Comment:
Could we add a bit more information here? A user might not know what tracing is. Particularly because we default this, I'd expect the API docs to tell us what the defaults are.

Reply:
done